includes/template-library/sources/local.php

```diff
@@ -1718,6 +1732,7 @@ private function remove_elementor_cpt_from_sitemap( array $post_types ) {
 	public function __construct() {
 		parent::__construct();
 
+		$this->app_services = new AppServices();
```

Comment:
Are you creating a new instance of the services for each usage?
 Don't you think it will be better to create the service once the general app.php is loaded and consume it from there?
 EDITED
 I saw you created the instances both in the JS and in PHP, but you didn't use it here, why?